### DISCUSSION TITLE
Block home feed (Distraction-Free YouTube)

### DISCUSSION THREAD

**Discussion by Fufec** [ReVanced/revanced-discussions#958] [Question] [answered]
Hi,
I think it would be really useful if there was an option in the layout settings to disable the home feed and redirect users to the subscriptions tab at startup. 

I believe that adding this feature would be a great improvement to ReVanced, and it would help users to avoid distractions with countless suggestions on the home feed.

Thank you for your time and effort.
Cheers!

**Comment by g-kartik** [comment]
I support this

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
The home feed tab can now be hidden, and the default start page can be set to the subscriptions tab.

Also enable `Always change start page` so it opens the different start page if the app is exited using the back button.

**Reply by asgunassekaran** [reply]
This works. Thanks.

**Reply by lakshyaagarwal242-lgtm** [reply]
How to do this i don't understand.

**Comment by Alebacce** [comment]
There's actually a "bug". If I implement for example the library section as preferred section on app opening, going back with a swipe (closing so the app) and reopening it, the app will open always on the feed section

**Reply by LisoUseInAIKyrios** [reply]
Update your patches.  There is a new setting to always change the start page and it solves that use case.

**Reply by Alebacce** [reply]
![Screenshot_20250428-114452.png](https://github.com/user-attachments/assets/0d4dce2a-da6e-4b84-88af-91244adc516d)

Is maybe this? Yes I've the patches updated. If the option Is this then there's the bug I've noticed, if not can you tell where can I find the right setting?

**Reply by LisoUseInAIKyrios** [reply]
You are using very old patches.  Update your app.

http://ReVanced.app

**Reply by Alebacce** [reply]
In the app It says patch are updated to 25/04/2025

**Reply by LisoUseInAIKyrios** [reply]
You have to repatch the app again.

**Reply by Alebacce** [reply]
Didn't know It, thanks!

**Reply by Alebacce** [reply]
Didn't know It, thanks!

**Comment by Rdkang** [comment]
I have encountered a case where it still shows the supposed to be bypassed homepage, 
if i navigate to the baseurl `youtube.com` or `m.youtube.com` in any browser then it will open youtube revanced with the homepage. 

Does anyone know a fix for this? @LisoUseInAIKyrios :)

I'm using the settings `Set start page` and `Always change start page`. And everything else works perfectly
